index.html

```diff
@@ -12,7 +12,7 @@
 <body class="navbar">
     <header>
         <ul>
-            <li>About</li>
+            <li>Abouts</li>
             <li>Work</li>
             <li>Contact</li>
         </ul>
```

Comment:
Well done with these good review comments. 

- [ ] However, as per the requirement you are expected to make at least two in-line comments.  But you have four-in-one comments and as such did not highlight the part of the code for each comment. Always highlight the section of the code that your comment concerns. So you would have had about four separate in-line comments. 
- [ ] Your first two comments were okay in terms of **_Politeness_**,  **_clarity_**, _**why**_ and _**what**._ . The **_why_** speaks to why you are suggesting the changes. **_What_** speaks to what should be done or what the issue is. **_Clarity_** speaks to how it is easy to understand without ambiguity. Politeness speaks about the manner in which the issue is communicated. 
- [ ] If you have more than one thing to say about an issue, please use bullet points for ease of assimilation. 

Please always ensure your comments point to these - What, why, Politeness and clarity. Please update the last two comments to reflect these points. **_Remember to keep them as separate in-line comments._**